src/realm/spec.cpp

```diff
@@ -84,7 +82,7 @@ void Spec::update_internals() noexcept
     m_num_public_columns = 0;
     size_t n = m_types.size();
     for (size_t i = 0; i < n; ++i) {
-        if (m_types.get(i) == col_type_BackLink) {
+        if (m_types.get(i) == int(col_type_BackLink)) {
```

Comment:
I would prefer `ColType(m_types.get(i)) == col_type_Backlink` in an effort to reduce the number of conversions from strongly typed to int (not blocking).